test/shared/get-dist-tag-version.test.ts

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import got from 'got';
+import { expect, use as chaiUse } from 'chai';
+import * as Sinon from 'sinon';
+import * as SinonChai from 'sinon-chai';
+import { stubMethod } from '@salesforce/ts-sinon';
+import { getDistTagVersion, DistTagJson } from '../../src/shared/get-dist-tag-version';
+
+chaiUse(SinonChai);
+
+describe('getReleaseNotes tests', () => {
+  let sandbox: sinon.SinonSandbox;
+  let gotStub: sinon.SinonStub;
+
+  let url;
+  let gotResponse: DistTagJson;
+
+  beforeEach(() => {
+    url = 'https://registry.npmjs.org/-/package/sfdx-cli/dist-tags';
+    gotResponse = {
+      latest: '1.2.3',
+      'latest-rc': '1.3.0',
+    };
+
+    sandbox = Sinon.createSandbox();
+
+    gotStub = stubMethod(sandbox, got, 'default');
+    gotStub.returns({
+      json: () => gotResponse,
+    });
+  });
+
+  afterEach(() => {
+    gotStub.restore();
```

Comment:
the `sandbox.restore` will do everything attached to the sandbox, so you shouldn't have to restore the stub.

Reply:
Ah good to know 👍